chapters/annotations.tex

```diff
@@ -888,6 +888,11 @@ \subsection{Component Instance}\label{component-instance}
 Sub-connectors in a hierarchical connector are only shown when they can be connected to.
 Non-connector components are only shown in the diagram layer.
 
+\subsubsection{Default Outline for Missing Graphics}\label{default-outline-for-missing-graphics}
+If the icon of a class does not contain any graphical primitives (including inherited ones, and regardless of visibility; but excluding connectors) it is said to be missing graphics.
+If the icon of a class used in a diagram is missing graphics then tools shall show a tool-dependent outline of the extent of the component to make improve usability.
```

Comment:
"make improve" should just be "improve" I guess?

Reply:
Yes, was trying to write two different variants.